packages/framework/src/Model/Administrator/Administrator.php

```diff
@@ -200,7 +214,14 @@ public function getLastActivity()
      */
     public function isSuperadmin()
     {
-        return $this->superadmin;
+        $isSuperAdmin = false;
+        foreach ($this->roles as $role) {
+            if ($role->getRole() === Roles::ROLE_SUPER_ADMIN) {
+                $isSuperAdmin = true;
```

Comment:
Ok, you are right. I always try/prefer to have only one `return` in method.  But in case of performance it makes sense to implement it this way. Fixed